bundle/Form/Builder/FormBuilder.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\InformationCollectionBundle\Form\Builder;
+
+use Ibexa\Contracts\Core\Repository\Values\Content\Location;
+use Ibexa\Contracts\Core\SiteAccess\ConfigResolverInterface;
+use Ibexa\Core\Repository\SiteAccessAware\ContentTypeService;
+use Netgen\Bundle\IbexaFormsBundle\Form\DataWrapper;
+use Netgen\Bundle\IbexaFormsBundle\Form\Payload\InformationCollectionStruct;
+use Netgen\Bundle\InformationCollectionBundle\Form\InformationCollectionUpdateType;
+use Netgen\InformationCollection\API\FieldHandler\CustomLegacyFieldHandlerInterface;
+use Netgen\InformationCollection\API\Value\Attribute;
+use Netgen\InformationCollection\API\Value\Collection;
+use Netgen\InformationCollection\API\Value\Legacy\FieldValue;
+use Netgen\InformationCollection\Core\Factory\FieldDataFactory;
+use Netgen\InformationCollection\Core\Persistence\FieldHandler\FieldHandlerRegistry;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\Routing\RouterInterface;
+
+class FormBuilder
+{
+    public function __construct(
+        protected readonly FormFactoryInterface $formFactory,
```

Comment:
`readonly` properties are allowed since PHP `8.1`, but the project also allows PHP `7.4`.

We should keep this and remove PHP `7.4` from `composer.json`, as we already have PHP `8.1` as minimum on other parts of stack (see https://github.com/netgen/media-site/blob/master/composer.json#L18).

Reply:
this was fixed in 0ff24026 commit